README.md

```diff
@@ -1 +1,41 @@
+# Deprecation Changelog Generator
 
+This tool generates the deprecation notice section of a changelog for a Silverstripe CMS release.
+
+It also lets you know if you need to perform follow-up actions (e.g. if some removed API didn't have a deprecation notice).
+
+**This tool is only intended for use by Silverstripe core committers or the Silverstripe Ltd CMS Squad**
+
+## Setup
+
+1. Clone this repo somewhere.
+1. Run `composer install` to install its dependencies.
+1. Set up a directory somewhere that this tool will use for cloning Silverstripe CMS into and for outputting any long-form data.
+
+## Usage
+
+If you're unsure of usage at any time, use the `--help` flag to get more details about any command, or run the `bin/deprecation-checker list` command to see all available commands.
+
+> [!NOTE]
+> All of the commands accept a `--dir` option. This determines the directory that will be used for all output and must be the same directory across all commands.
+> If this flag is ommitted, the current working directory will be used.
+
+1. Run `bin/deprecation-checker clone <recipe> <fromConstraint> <toConstraint>`
+    - For example `bin/deprecation-checker clone sink 5.4.x-dev 6.0.x-dev --dir=~/dump/deprecation-checks`
```

Comment:
`sink`

Need to document CloneCommand::RECIPE_SHORTCUTS in the readme, or show the command to list the shortcuts in the readme

Reply:
I kinda feel like "If you're unsure of usage at any time, use the `--help` flag to get more details about any command" covers this - but I'll add an explicit note about it.